web/server/codechecker_server/api/report_server.py

```diff
@@ -1304,6 +1316,12 @@ def getDiffResultsHash(self, run_ids, report_hashes, diff_type,
                 if run_ids:
                     results = results.filter(Report.run_id.in_(run_ids))
 
+                if tag_ids:
+                    results = results.outerjoin(
+                        RunHistory, RunHistory.run_id == Report.run_id) \
```

Comment:
Could you please use one of the following form to break lines:

```suggestion
                    results = results \
                        .outerjoin(RunHistory, RunHistory.run_id == Report.run_id) \
```

```suggestion
                    results = results.outerjoin(RunHistory,
                                                RunHistory.run_id == Report.run_id) \
```
We use these forms in other places as well.